### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by mctaylors**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+![DamageVignette Dark](https://user-images.githubusercontent.com/95250141/215292364-d9714cd2-a8fd-4503-abdf-6b6408fd0f41.png#gh-dark-mode-only)
+![DamageVignette Light](https://user-images.githubusercontent.com/95250141/215295583-e1093205-00cf-45f3-8fa2-66baa41245a8.png#gh-light-mode-only)
```
Comment (Octol1ttle) [inline README.md]: image
text overlaps

File: README.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+
+![License](https://img.shields.io/github/license/Octol1ttle/DamageVignette-Fabric)
+![GitHub Latest Release](https://img.shields.io/github/v/release/Octol1ttle/DamageVignette-Fabric)
+![Latest Release Date](https://img.shields.io/github/release-date/Octol1ttle/DamageVignette-Fabric)
+![Game Versions](https://img.shields.io/modrinth/game-versions/damagevignette)
+
+Adds an overlay (similar to getting close to world border) when low on health
+
+![Fabric](https://user-images.githubusercontent.com/95250141/215292867-42f6f2dc-c26e-491e-963e-dad9dfce636b.png)
```
Comment (Octol1ttle) [inline README.md:9]: put this above the summary

File: README.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+## About this mod
+
+This mod will show a red vignette effect when your HP gets low to alert you. That's it.
+
+You can define health thresholds for vignette intensity if you have Cloth Config and Mod Menu installed. Compatible with Iris Shaders.
+
+**Note: Please make sure your graphics settings allow vignette to be displayed! With Sodium, it is a separate option that you need to enable, otherwise set your graphics to "Fancy"**
+
+## Downloads
+![GitHub Total Downloads](https://img.shields.io/github/downloads/Octol1ttle/DamageVignette-Fabric/total?logo=github)
```
Comment (Octol1ttle) [inline README.md]: useless

**Review by Octol1ttle — CHANGES_REQUESTED** [review]
my man boutta make readmes for every repo of mine

File: README.md (hunk 3 above)
Comment (makuraslvt) [reply README.md]: useless what actually?
downloads or note

Comment (Octol1ttle) [reply README.md]: GitHub downloads counter, keep the Modrinth one

Comment (makuraslvt) [reply README.md]: then highlight only line 23 lol

Comment (Octol1ttle) [reply README.md]: literally what I did, blame github

**Review by Octol1ttle — APPROVED** [review]

**Pull request merged**